src/components/menus/Items.ts

```diff
@@ -114,6 +115,29 @@ export class Items<TGameStartr extends FullScreenPokemon> extends Component<TGam
         const options: any[] = [
             {
                 callback: (): void => {
+                    const category = this.gameStarter.itemsHolder.getItem(listing.item).category;
+                    if (category === "HM") {
```

Comment:
Don't throw these errors. If you need to provide an effect, just have a `(): void => {}` that does nothing.